README.md

```diff
@@ -162,6 +162,35 @@ In case you need to assign a specific port to a Bypass instance to listen on, yo
 bypass = Bypass.open(port: 1234)
 ```
 
+## How to use with ESpec
+
+While Bypass primary targets ExUnit, as it's the official Elixir builtin test framework, it can also be used with [ESpec](https://hex.pm/packages/espec). The test configuration is basically the same, and the only difference is that you must explicitly call the function to verify the expectations in the `finally` block.
+
+```elixir
+defmodule TwitterClientSpec do
+  use ESpec, async: true
+
+  before do
+    bypass = Bypass.open
```

Comment:
I think in your code example this would have to be `Bypass.open(:espec)` no?

Can you make this an application configuration instead?